docs/python/editing.md

```diff
@@ -86,20 +86,35 @@ While editing, you can right-click different identifiers to take advantage of se
+
+![Add import code action](images/editing/quickFix.png)
 
 This Code Action also recognizes some of the popular abbreviations for the following common Python packages: `numpy` as np, `tensorflow` as tf, `pandas` as pd, `matplotlib.pyplot` as plt, `matplotlib` as mpl, `math` as m, `scipi.io` as spio, and `scipy` as sp, `panel` as pn, and `holoviews` as hv.
 
 ![Common package abbreviations](images/editing/packageAbbreviations.gif)
 
-The import suggestions list is ordered with import statements for packages (or modules) at the top. It will also include statements for more modules and/or members (classes, objects, etc.) from specified packages.
+The import suggestions list displays the top 3 high-confidence import options, prioritized based on: most recently used imports, symbols from the same module, symbols from the standard library, symbols from user modules, symbols from third-party packages, and finally sorting by module and symbol name.
+
+### Search for additional import matches
+By default, the add import Quick Fix only shows 3 high-confidence import options. If they don't list what you are looking for, you can use the Pylance **Search for additional import matches** Quick Fix for missing import errors. This Quick Fix displays a quick pick menu that enables you to search for import options that prefix-match the missing import symbol.
+
+![Search for additional import matches Code Action](images/editing/search-imports-code-action.gif)
+
 
-Just like with auto imports, only top-levels symbols are suggested by default. You can customize this behavior through the `python.analysis.packageIndexDepths` setting.
+### Change spelling
+Pylance displays the **Change spelling** Quick Fix on unresolved variables or missing imports diagnostics when they are likely caused by typos. This Code Action suggests the correct spelling of the symbol, based on the closest matches found in the workspace.
 
-## Refactoring
+![Change spelling code action on missing import due to a typo](images/editing/change-spelling-code-action.gif)
 
-The Python extension adds the following refactoring functionalities: **Extract Variable**, **Extract Method** and **Rename Module**. It also supports extensions that implement additional refactoring features such as **Sort Imports**.
+
+> **Note**: Symbol aliases defined in user code (for example,  the `bar` alias in `import foo as bar`) are not supported by these Quick Fixes for performance reasons. Just like with auto imports, only top-level symbols are suggested by default, but you can customize this behavior for specific packages through the `python.analysis.packageIndexDepths` setting.
+
+## Refactorings
+
+The Python extension adds the following refactoring functionalities via the Pylance extension: **Extract Variable**, **Extract Method**, **Rename Module**, **Move Symbol** and **Implement All Inherited Abstract Classes**. It also supports extensions that implement additional refactoring features, such as **Sort Imports**.
```

Comment:
It might be nice to provide a link somewhere in the last sentence to the ### Sort Imports section section later in this doc that talks about the extensions that provide this Code Action.

Reply:
in this case this paragraph is just mentioning all sub sections in the refactoring section. Once the page is published we can revisit this and add al links in case we think it'll make it easier to navigate between them :)